issues_test.go

```
@@ -93,15 +93,15 @@ func TestIssues_20(t *testing.T) {
 // expected: 10s左右执行完所有任务
 // actual: 执行了 100s左右
 func TestIssues_61(t *testing.T) {
-	var em = event.NewManager("default", func(o *event.Options) {
+	var em = event.NewManager[event.M]("default", func(o *event.Options) {
 		o.ConsumerNum = 10
 		o.EnableLock = false
 	})
 	defer em.CloseWait()
```

golangci-lint (1.64.8)

Error return value of `em.CloseWait` is not checked

(errcheck)